homeassistant/components/lifx/manager.py

```diff
@@ -375,6 +409,27 @@ async def start_effect(
             )
             await self.effects_conductor.start(effect, bulbs)
 
+        elif service == SERVICE_EFFECT_SKY:
+            await asyncio.gather(
+                *(
+                    coordinator.async_set_matrix_effect(
+                        effect=EFFECT_SKY,
+                        speed=kwargs.get(ATTR_SPEED, EFFECT_SKY_DEFAULT_SPEED),
+                        sky_type=kwargs.get(ATTR_SKY_TYPE, EFFECT_SKY_DEFAULT_SKY_TYPE),
+                        cloud_saturation_min=kwargs.get(
+                            ATTR_CLOUD_SATURATION_MIN,
+                            EFFECT_SKY_DEFAULT_CLOUD_SATURATION_MIN,
+                        ),
+                        cloud_saturation_max=kwargs.get(
+                            ATTR_CLOUD_SATURATION_MAX,
+                            EFFECT_SKY_DEFAULT_CLOUD_SATURATION_MAX,
+                        ),
+                        palette=kwargs.get(ATTR_PALETTE, None),
```

Comment:
Would be nicer to build local vars for the args that do not change inside the loop instead of doing the same get each loop